Editor/Window/ConnectToFleetInput.cs

```diff
@@ -163,8 +167,10 @@ private async Task UpdateFleetMenu()
                     _fleetAttributes = new List<FleetAttributes>();
                 }
 
-                _fleetNameDropdownContainer.choices = _fleetAttributes.Select(fleet => fleet.Name).ToList();
-                _fleetNameDropdownContainer.value = _stateManager.AnywhereFleetName;
+                _fleetNameDropdownContainer.choices =
+                    _fleetAttributes.Select(fleet => $"{fleet.Name} | {fleet.FleetId}").ToList();
+                _fleetNameDropdownContainer.value =
+                    $"{_stateManager.AnywhereFleetName} | {_stateManager.AnywhereFleetId}";
                 _fleetIdText.text = _stateManager.AnywhereFleetId;
 
                 var fleet = _fleetAttributes.FirstOrDefault(fleet => fleet.Name == _stateManager.AnywhereFleetName);
```

Comment:
We need to update this and any other location that uses fleet name for searching to use the fleet id; otherwise, none of the code will work for duplicate names when you select the second entry of a name in the list

Reply:
Missed this instance